3rdparty/ippicv/ippicv.cmake

```diff
@@ -13,12 +18,24 @@ function(download_ippicv root_var)
   elseif((UNIX AND NOT ANDROID) OR (UNIX AND ANDROID_ABI MATCHES "x86"))
     set(OPENCV_ICV_PLATFORM "linux")
     set(OPENCV_ICV_PACKAGE_SUBDIR "ippicv_lnx")
+    # Android NDK issue https://github.com/opencv/opencv/issues/26072
     if(X86_64)
-      set(OPENCV_ICV_NAME "ippicv_2021.12.0_lnx_intel64_20240425_general.tgz")
-      set(OPENCV_ICV_HASH "d06e6d44ece88f7f17a6cd9216761186")
+      if(ANDROID)
+        set(OPENCV_ICV_NAME "ippicv_2021.10.1_lnx_intel64_20231206_general.tgz")
+        set(OPENCV_ICV_HASH "90884d3b9508f31f6a154165591b8b0b")
+      else()
+        set(OPENCV_ICV_NAME "ippicv_2021.12.0_lnx_intel64_20240425_general.tgz")
+        set(OPENCV_ICV_HASH "d06e6d44ece88f7f17a6cd9216761186")
+      endif()
```

Comment:
Is the workaround for `x86_64` necessary? It seems that the issue #26072 exists only for `x86`, doesn't it?